src/backend/web/handlers/team.py

```diff
@@ -119,3 +122,27 @@ def team_list(page: int) -> str:
         "current_page": page,
     }
     return render_template("team_list.html", template_values)
+
+
+@cached_public
+def avatar_list(year: Optional[Year] = None) -> Response:
+    year = year or SeasonHelper.get_current_season()
+
+    valid_years = list(range(2018, SeasonHelper.get_max_year() + 1))
+    valid_years.remove(2021)  # No avatars in 2021 :(
+
+    if year not in valid_years:
+        abort(404)
+
+    avatars_future = Media.query(Media.media_type_enum
+                                 == MediaType.AVATAR, Media.year == year).fetch_async()
+    avatars = sorted(avatars_future.get_result(),
+                     key=lambda a: int(a.references[0].id()[3:]))
```

Comment:
This `avatar.references[0].id()[3:]` bit of code is used in both the template and here. Maybe we throw this as a helper on the `Media` class and DRY all of this code?